tests/unit/base/trained_model_serialization_unittest.cc.py

```diff
@@ -6,20 +6,30 @@
 # (at your option) any later version.
 #
 
-# Classes that need custom initialization
-IGNORE_MACHINES = {
-    'CLinearMachine' : [
-        'CFeatureBlockLogisticRegression', 'CLibLinearMTL',
-        'CMultitaskLinearMachine', 'CMultitaskLogisticRegression',
-        'CMultitaskL12LogisticRegression', 'CMultitaskLeastSquaresRegression',
-        'CMultitaskTraceLogisticRegression', 'CMultitaskClusteredLogisticRegression',
-        'CLatentSVM', 'CLatentSOSVM', 'CDomainAdaptationSVMLinear'
-    ],
-    'CKernelMachine' : [
-        'CDomainAdaptationSVM', 'CMKLRegression',
-        'CMKLClassification', 'CMKLOneClass', 'CSVM'
-    ]
-}
+# Ignore classes that need custom initialization and:
+# (Multiclass)SVM: doesn't implement a solver
+# ScatterSVM, MulticlassLibSVM: error C <= 0
```

Comment:
comments should probably be put begind the lines with the individual classes below rather than all stacked up on top